Drumstick.dmidiplayer version 1.6.0

pedrolcl opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60988)

pedrolcl committed:
Drumstick.dmidiplayer version 1.6.0

ItzLevvie commented on manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.installer.yaml:
```diff
@@ -0,0 +1,28 @@
+ReleaseDate: 2022-05-17
+Installers:
+- Architecture: x64
+  InstallerUrl: http://downloads.sourceforge.net/project/dmidiplayer/v1.6.0/dmidiplayer-1.6.0-win-x64-setup.exe
```
```suggestion
  InstallerUrl: https://sourceforge.net/projects/dmidiplayer/files/v1.6.0/dmidiplayer-1.6.0-win-x64-setup.exe/download
```
for consistency with other sourceforge packages.

ItzLevvie commented on manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml:
```diff
@@ -0,0 +1,32 @@
+PublisherSupportUrl: https://sourceforge.net/p/dmidiplayer/tickets/
+# PrivacyUrl: 
+Author: Pedro López-Cabanillas <plcl@users.sourceforge.net>
+PackageName: Drumstick Multiplatform MIDI File Player
```
```suggestion
PackageName: dmidiplayer
```
This must match Control Panel.

pedrolcl replied:
OK

ItzLevvie commented on manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml:
```diff
@@ -0,0 +1,32 @@
+PackageIdentifier: Drumstick.dmidiplayer
+PackageVersion: 1.6.0
+PackageLocale: en-US
+Publisher: Drumstick Projects
```
```suggestion
Publisher: Drumstick
```
This must match Control Panel.

ItzLevvie commented on manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.locale.en-US.yaml:
```diff
@@ -0,0 +1,32 @@
+Author: Pedro López-Cabanillas <plcl@users.sourceforge.net>
+PackageName: Drumstick Multiplatform MIDI File Player
+PackageUrl: https://sourceforge.net/projects/dmidiplayer/
+License: GPLv3
```
```suggestion
License: GPL-3.0-only
```

pedrolcl replied:
I've changed the License field to the correct SPDX short identifier, but not the suggested one.

pedrolcl committed:
ItzLevvie suggestions

InstallerUrl changed
Publisher changed
PackageName changed
License changed to the correct SPDX short identifier

JohnMcPMS commented:
@wingetbot waivers Add Validation-Domain
